YoutubeExplode/YoutubeClient.cs

```diff
@@ -302,5 +340,56 @@ public static string ParseVideoId(string videoUrl)
 
             throw new FormatException("Could not parse Video ID from given string");
         }
+
+        /// <summary>
+        /// Verifies that the given string is a valid youtube playlist ID
+        /// </summary>
+        /// <returns>True if valid, false otherwise</returns>
+        public static bool ValidatePlaylistId(string playlistId)
+        {
+            if (playlistId.IsBlank())
+                return false;
+
+            return !Regex.IsMatch(playlistId, @"[^0-9a-zA-Z_\-]");
+        }
+
+        /// <summary>
+        /// Tries to parse playlist ID from a youtube playlist URL
+        /// </summary>
+        /// <returns>Whether the execution was successful or not</returns>
+        public static bool TryParsePlaylistId(string playlistUrl, out string playlistId)
+        {
+            playlistId = default(string);
+
+            if (playlistUrl.IsBlank())
+                return false;
+
+            // https://www.youtube.com/playlist?list=PLOU2XLYxmsIJGErt5rrCqaSGTMyyqNt2H
+            var regularMatch = Regex.Match(playlistUrl, @"youtube\..+?/playlist\?.*?list=(.+?)(?:&|$)").Groups[1].Value;
```

Comment:
This link appears to represent a play list too:

https://www.youtube.com/watch?v=b8m9zhNAgKs&list=PL9tY0BWXOZFuFEG_GtOBZ8-8wbkH-NVAr